great_expectations/experimental/metric_repository/metric_retriever.py

```diff
@@ -293,7 +294,7 @@ def _get_table_columns(self, batch_request: BatchRequest) -> Metric:
     def _get_table_column_types(self, batch_request: BatchRequest) -> Metric:
         metric_name = MetricTypes.TABLE_COLUMN_TYPES
 
-        metric_lookup_key: _MetricKey = (metric_name, tuple(), "include_nested=True")
+        metric_lookup_key = MetricConfigurationID(metric_name, tuple(), tuple())
```

Comment:
This `include_nested=True` is the only reference to a mutated `MetricConfiguration.id` I could find in the code. Everywhere else this argument gets updated is in test mock responses. At instantiation time the 3rd argument to this id is an empty tuple, but later `metric_value_kwargs` gets updated and that reference mutates the id. Now the id is no longer mutated and we needed to update this lookup value.